Packs/DeveloperTools/Integrations/XsoarServerPowershell/README.md

```diff
@@ -0,0 +1,78 @@
+This integration allows interacting with XSOAR server, mostly for internal use.
```

Comment:
@GalRabin let's remove this file. No need for documentation in the site for this integration.

Reply:
Done